modules/overrides/interface.go

```diff
@@ -68,5 +71,7 @@ type Interface interface {
 
 	// Management API
 	WriteStatusRuntimeConfig(w io.Writer, r *http.Request) error
-	WriteTenantOverrides(w io.Writer, r *http.Request, tenant string) error
+
+	TenantsHandler(w http.ResponseWriter, req *http.Request)
```

Comment:
I'm wondering if these should not be in `Interface`.  It seems like they are an aspect of the hosting in `/cmd/tempo/` and not part of the overrides modules directly.  Let's chat offline to help me get up to speed on the interaction between overrides.

Reply:
My reply earlier

> Yeah it’s an unfortunate consequence from hiding the runtime and user-config overrides behind an interface. The only public API exposed in the overrides package is `Interface`. When you setup the overrides, we return either a `runtimeConfigOverridesManager` or a `userConfigurableOverridesManager` based upon your configuration.
> 
> The reason there are two handlers for each page is we have a version with runtime overrides only and one with runtime+user-config overrides.
> We could also try to extend `Interface` to pass the config but that would be difficult to do generically I think.
> 
> By moving the status page into the overrides module we can access internal data more easily and it kind of makes sense that the module and its management page are tied closer together.

I actually just thought of an implementation that doesn't use two implementations, let me try it out.